### PR TITLE
Fix warning on VS2017 [19882]

### PR DESCRIPTION
**Pull request by richiware**

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Linux/325/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Mac/239/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Windows/324/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Linux/326/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Mac/240/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Windows/325/)

File: fastcdr.repos
Comment (JLBuenoLopez-eProsima) [inline fastcdr.repos:1]: Please, revert changes in this file.

**Review by JLBuenoLopez-eProsima — CHANGES_REQUESTED** [review]
Changes LGTM though there is one file whose changes need to be reverted. Let us wait for the CI results and as the changes does not affect the CI, we can proceed and merge.

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Linux/327/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Mac/241/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Windows/326/)

**Review by JLBuenoLopez-eProsima — APPROVED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Linux/328/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Mac/242/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastCDR%20Manual%20Windows/327/)

**Pull request merged**